### DISCUSSION TITLE
How can I remove or reset annotations ?

### DISCUSSION THREAD

**Discussion by DewenHona** [google/model-viewer#2498] [Q&A] [answered]
I am working on  model-viewer project

In model-viewer tag I have written a function which display models after selecting them using buttons.
How can I add hotspots specific to a particular model in the same <model-viewer?

Thank You 😀

**Comment by elalish** [comment, marked as answer]
Annotations are just HTML elements, so you can add and remove them as children of the model-viewer element with JS like any other elements in the DOM tree.
